api/app/models/spree/webhooks/endpoint.rb

```diff
@@ -0,0 +1,7 @@
+module Spree
+  module Webhooks
+    class Endpoint < Spree::Base
```

Comment:
@sebastian-palma I suppose we need some data validation here, URL presence & format

Reply:
Do you have in mind an effective way to validate the URL format @damianlegawiec? I saw there are a couple of other models with URL columns, but they don't have a format validation.

Reply:
@sebastian-palma https://coderwall.com/p/ztig5g/validate-urls-in-rails seems to be the best way, no additional dependencies needed